### PR TITLE
i18n localization, externalize user displayed messages to properties file

### LINKED ISSUE #67: Set up globalization framework
Setup framework and, if time, externalize existing strings (otherwise, open separate issue for externalizing strings)

### PR DESCRIPTION
**Pull request by kathrynkodama**
Fixes #67 
Signed-off-by: Kathryn Kodama <kathryn.s.kodama@gmail.com>

### CODE REVIEW

**Comment by kathrynkodama** [comment]
PR to follow to add the copyright header to all of the outstanding Java files

File: src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java [hunk 1]
```diff
@@ -12,8 +21,11 @@
 import io.openliberty.tools.intellij.LibertyPluginIcons;
 import io.openliberty.tools.intellij.util.Constants;
 import io.openliberty.tools.intellij.util.LibertyProjectUtil;
+import io.openliberty.tools.intellij.util.LocalizedResourceUtil;
 import org.jetbrains.annotations.NotNull;
 
+import java.text.MessageFormat;
```
Comment (turkeylurkey) [inline src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java:27]: Is this import needed here and 5 other files or is it left over from intermediate work?

File: src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java [hunk 2]
```diff
@@ -27,15 +39,15 @@ public class LibertyGeneralAction extends AnAction {
...
+            String msg = LocalizedResourceUtil.getMessage("liberty.project.does.not.resolve", actionCmd);
             notifyError(msg);
             log.debug(msg);
             return;
         }
 
         buildFile = (VirtualFile) e.getDataContext().getData(Constants.LIBERTY_BUILD_FILE);
         if (buildFile == null) {
-            String msg = "Unable to " + actionCmd + ": Could not resolve configuration file for  " + project.getName() + ". Ensure you run the Liberty action from the Liberty tool window";
+            String msg = LocalizedResourceUtil.getMessage("liberty.build.file.does.not.resolve", actionCmd,project.getName());
```
Comment (turkeylurkey) [inline src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java:50]: At some point someone is going to want to put a space after the comma "Cmd,proj"

File: src/main/resources/messages/LibertyBundles.properties [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+action.io.openliberty.tools.intellij.actions.RefreshLibertyToolbar.description=Refresh Liberty tool window projects
+action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.text=Run Liberty action
+action.io.openliberty.tools.intellij.actions.RunLibertyDevTask.description=Run the selected Liberty action
+
+# Messages for Liberty actions and pop-up dialog
+no.liberty.projects.detected=No Liberty Maven or Liberty Gradle projects detected in this workspace.
+liberty.dev.not.started.notification.title=Liberty dev mode has not been started
+liberty.dev.not.started.notification.content=Liberty dev mode has not been started on {0}. \nStart Liberty dev mode from the Liberty tool window.
+liberty.project.does.not.resolve=Unable to {0}: Could not resolve project. Ensure you run the Liberty action from the Liberty tool window
+liberty.build.file.does.not.resolve=Unable to {0}: Could not resolve build file for  {1}. Ensure you run the Liberty action from the Liberty tool window
```
Comment (turkeylurkey) [inline src/main/resources/messages/LibertyBundles.properties:42]: Sometimes two blanks can look weird "for  {1}".

**Review by turkeylurkey — APPROVED** [review]

**Pull request merged**